src/EnergyPlus/HeatRecovery.cc

```diff
@@ -2498,6 +2486,9 @@ namespace HeatRecovery {
         if ((EconomizerActiveFlag || HighHumCtrlActiveFlag) && this->EconoLockOut) UnitOn = false;
 
         if (UnitOn) {
+            constexpr const char *ThisSubTSat("CalcDesiccantBalancedHeatExch:   TSat");
```

Comment:
`std::string_view`. `const char *` is not memory safe, even in DEBUG.

Reply:
constexpr const? I used constexpr std::string_view.